vars/mtaBuild.groovy

```diff
@@ -131,7 +133,12 @@ void call(Map parameters = [:]) {
 
             def mtaCall
             def options = []
-            options.push("--mtar ${id}.mtar")
+
+            String mtarName = configuration.mtarName?.trim()
+            if (!mtarName) {
+                mtarName = id
```

Comment:
As `getMtaId(mtaYamlName)` requires a file operation I would suggest to move it here to avoid it if a name is given.
```suggestion
                mtarName = getMtaId(mtaYamlName)
```

Reply:
👍